chain/reward.go

```diff
@@ -0,0 +1,30 @@
+package chain
+
+import (
+	"github.com/nknorg/nkn/common"
+	"github.com/nknorg/nkn/util/config"
+)
+
+var (
+	MiningRewards = make(map[uint32]common.Fixed64)
+)
+
+func init() {
+	for i := uint32(0); i < config.TotalRewardDuration; i++ {
+		MiningRewards[i] = common.Fixed64(config.InitialReward - config.ReductionAmount*int(i))
```

Comment:
We should probably verify that config.InitialReward > config.ReductionAmount*int(i) in case the parameter is wrong